src/commandline_frame.ts

```diff
@@ -318,7 +321,7 @@ function applyWithTmpTextArea(fn) {
 
 /** @hidden **/
 export async function setClipboard(content: string) {
-    await Messaging.messageOwnTab("commandline_content", "focus")
+    await Messaging.messageOwnTab<Messages.CommandlineContent>()("commandline_content", "focus", [])
```

Comment:
I don't like the empty array at the end, what's it for?

The `CommandlineContent` and `commandline_content` repetition is a little worrying too :)